worldengine/generation.py

```diff
@@ -212,7 +213,7 @@ def generate_world(w, step):
 
     # FIXME: create setters
     IrrigationSimulation().execute(w, seed)
-    TemperatureSimulation().execute(w, seed)
+    
```

Comment:
Once you rebase to master, these changes will have to be redone. But swapping out two lines shouldn't be an insurmountable task.^^